app/components/display/search_result_component.rb

```diff
@@ -1,5 +1,6 @@
 class Display::SearchResultComponent < ViewComponent::Base
   include ModalHelper
+  include ApplicationHelper
```

Comment:
don't include an application helper, it is too large and container too many things, so we don't know its usage here, 
if needed in the component create a helper for the translation only.